commit ae4a380109d154ffbc7e2b6204d79b1a171671bb
Author: Kees Cook <kees@kernel.org>
Date:   Sat Oct 10 23:16:27 2020 -0700

    MAINTAINERS: change hardening mailing list
    
    As more email from git history gets aimed at the OpenWall
    kernel-hardening@ list, there has been a desire to separate "new topics"
    from "on-going" work.
    
    To handle this, the superset of hardening email topics are now to be
    directed to linux-hardening@vger.kernel.org.
    
    Update the MAINTAINERS file and the .mailmap to accomplish this, so that
    linux-hardening@ can be treated like any other regular upstream kernel
    development list.
    
    Signed-off-by: Kees Cook <keescook@chromium.org>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Cc: Randy Dunlap <rdunlap@infradead.org>
    Cc: Emese Revfy <re.emese@gmail.com>
    Cc: "Tobin C. Harding" <me@tobin.cc>
    Cc: Tycho Andersen <tycho@tycho.pizza>
    Cc: Jonathan Corbet <corbet@lwn.net>
    Link: https://lore.kernel.org/linux-hardening/202010051443.279CC265D@keescook/
    Link: https://lkml.kernel.org/r/20201006000012.2768958-1-keescook@chromium.org
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index e83aee101b66..25e60fce7aeb 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7240,7 +7240,7 @@ F:	drivers/staging/gasket/
 GCC PLUGINS
 M:	Kees Cook <keescook@chromium.org>
 R:	Emese Revfy <re.emese@gmail.com>
-L:	kernel-hardening@lists.openwall.com
+L:	linux-hardening@vger.kernel.org
 S:	Maintained
 F:	Documentation/kbuild/gcc-plugins.rst
 F:	scripts/Makefile.gcc-plugins
@@ -9802,7 +9802,7 @@ F:	drivers/scsi/53c700*
 LEAKING_ADDRESSES
 M:	Tobin C. Harding <me@tobin.cc>
 M:	Tycho Andersen <tycho@tycho.pizza>
-L:	kernel-hardening@lists.openwall.com
+L:	linux-hardening@vger.kernel.org
 S:	Maintained
 T:	git git://git.kernel.org/pub/scm/linux/kernel/git/tobin/leaks.git
 F:	scripts/leaking_addresses.pl